system/View/Parser.php

```diff
@@ -103,7 +103,7 @@ public function render(string $view, ?array $options = null, ?bool $saveData = n
         }
 
         $fileExt = pathinfo($view, PATHINFO_EXTENSION);
-        $view    = empty($fileExt) ? $view . '.php' : $view; // allow Views as .html, .tpl, etc (from CI3)
+        $view    = ($fileExt === '') ? $view . '.php' : $view; // allow Views as .html, .tpl, etc (from CI3)
```

Comment:
Same here... is it supposed to increase readability? For me, it's the opposite.

Reply:
In this case, yes.